docker/Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM scientificlinux/sl:7
+FROM fermilab/fnal-wn-sl7
 
 RUN yum -y update && \
     yum -y install epel-release && \
-    yum -y install sudo screen wget patch libtool git-all gcc gcc-c++ gcc-gfortran \
```

Comment:
sudo will not work under singularity.
We should also consider removing screen. As screen should be invoke outside of the container

Reply:
Agree.  You can remove screen also.